pyobjus/pyobjus.pyx

```diff
@@ -118,12 +128,39 @@ def selector(name):
     Returns:
         ObjcSelector instance, which contains SEL pointer
     """
+    cdef bytes c_name
+    if isinstance(name, bytes):
+        c_name = name
+    else:
+        c_name = name.encode("utf8")
```

Comment:
import unicode_literals from future maybe?

Reply:
It's apparently not often a good idea https://python-future.org/unicode_literals.html